server/save.py

```diff
@@ -330,6 +330,15 @@ def upgrade(version):
             )
         db.foreign_keys = True
         Constants.get().update(save_version=Constants.save_version + 1).execute()
+    elif version == 23:
```

Comment:
You also need to increase the SAVE_VERSION at the top of this file, or the migration will never happen

Reply:
done

Reply:
SAVE_VERSION needs to be +1 from the last elif? 
i.e. SAVE_VERSION = 24
Done.....

Reply:
Yes on startup the server checks if a migration is required based on the SAVE_VERSION :)